source/type/pb_converter.cpp

```diff
@@ -1946,6 +1946,7 @@ pbnavitia::Metadatas* PbCreator::mutable_metadatas(){
 
 void PbCreator::clear_feed_publishers(){
     response.clear_feed_publishers();
+    contributors.clear();
```

Comment:
Not sure about a possible double use with: https://github.com/CanalTP/navitia/blob/dev/source/type/pb_converter.cpp#L1777